test/integration/targets/docker_host_facts/tasks/test_host_facts.yml

```diff
@@ -0,0 +1,235 @@
+---
+- name: Create random container name prefix
+  set_fact:
+    cname_prefix: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
```

Comment:
Why not define `cname` directly? If you don't need more than one name, there's no sense of having a prefix and then defining a name out of it.

Reply:
Because if you run an integration test on a host running docker already and use a simple name as "test" by example, you may already have such an object. Random names are in integration tests for `docker_container` or `docker_volume` integration tests, I see nothing wrong in it, just a safety precaution.

Reply:
directly == don't first define `cname_prefix` to use it exactly once for `cname`, but instead define `cname` here :)